ess/src/API/EMBC.ESS.Shared.Contracts/Submissions/Search.cs

```diff
@@ -53,6 +53,7 @@ public class RegistrantsSearchCriteria : SearchCriteria
         public string LastName { get; set; }
         public string DateOfBirth { get; set; }
         public bool IncludeRestrictedAccess { get; set; }
+        public bool LeaveSecurityAnswersUnmasked { get; set; } = false;
```

Comment:
I don't think this is needed - outside of the backend service, answers should always be masked